filters/log/log.go

```diff
@@ -27,6 +29,13 @@ const (
 	// reject reason information into the state bag to pass the
 	// information to the auditLog filter.
 	AuthRejectReasonKey = "auth-reject-reason"
+	// UnverifiedAuditLogName is th filtername seend by the user
+	UnverifiedAuditLogName = "unverifiedAuditLog"
+
+	UnverifiedAuditHeader = "X-Unverified-Audit"
```

Comment:
@aryszka what do you think about this idea?
Passing this information by header to the accessLog.

Reply:
from the two alternatives that we discussed today (globally enabling this access log field or allowing it to be controlled in the routes), i find this approach better on the long term.

I still have the concern about the naming, but right now I don't have a better name, either. The reason why I don't like the naming is that it doesn't call the thing what it is but it refers to a concept that it can be used for ("using the sub claim from the auth header for auditing" instead of just "printing the sub claim from the auth header"). Let's park this for a day, maybe someone can give it a good name. @hjacobs do you have a suggestion for a good name for this?